tests/foreman/ui/test_eol_banner.py

```diff
@@ -4,17 +4,12 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
-
 :CaseComponent: Dashboard
 
 :Team: Endeavour
 
-:TestType: Functional
```

Comment:
@vijaysawant Could you keep a separate commit for fixing this please?

Reply:
OR open a separate PR instead as this eol banner tests added by @pondrejk are only applicable for master and 6.15.z, and for your PR to cherrypick to below versions auto-cherrypick workflow will fail because of this :)